docs/src/ReactPlayground.js

```diff
@@ -6,6 +6,8 @@ const classNames = require('classnames');
 const React = require('react');
 const ReactDOM = require('react-dom');
 
+const bootstrapUtils = require('../../src/utils/bootstrapUtils.js').default;
```

Comment:
Hi @taion I noticed this is merged but I did not make any changes to this import statement. Would you like me to submit an additional PR with any changes to it?